faq/doc-copy.php

```diff
@@ -26,10 +23,10 @@
     Many people consider this very boring stuff but it is important that we get it
     right. <i>Thanks for proofreading it!</i>
 </p>
-<table border="0" cellspacing="0" width="100%">
+<table style="width: 100%">
```

Comment:
Oh geeze. Let's just replace this whole table with a simple `<hr>`.

Reply:
There are lots of places we use tables for layout that we probably shouldn't, but that's a different day.